src/network/config.js

```diff
@@ -7,7 +7,12 @@ let {isObject, clone} = require('lodash/lang');
  */
 let configuration = {
   CORS: true,
-  xhrErrors: {}
+  xhrErrors: {},
```

Comment:
I don't think it should be the same object. 
This config is for every request, the config for the rate limiter should be separated, except enableRateLimiter, which is indeed config for a request